projectq/meta/_control.py

```diff
@@ -24,18 +24,24 @@
 """
 
 from projectq.cengines import BasicEngine
-from projectq.meta import ComputeTag, UncomputeTag
+from projectq.meta import ComputeTag, UncomputeTag, Compute, Uncompute
 from projectq.ops import ClassicalInstructionGate
 from projectq.types import BasicQubit
+from projectq.ops import _command, X
 from ._util import insert_engine, drop_engine_after
+from enum import Enum
+
+class State(Enum):
```

Comment:
```suggestion
class State(IntEnum):
```

This would allow you to avoid needing to write `State.value` explicitly in the code.